routes/api/upgrade/prechecks.js

```diff
@@ -9,10 +9,11 @@ router.get('/', function(req, res) {
         res.status(500).json({'errors': errors});
     } else {
         res.status(200).json({
```

Comment:
Can we get a rotating response that returns passed for half of the checks on the first try, then returns true for all? its much nicer to fully test the response locally.